--- a/configure.ac
+++ b/configure.ac
@@ -751,8 +751,6 @@ AC_CONFIG_FILES([Makefile
 	contrib/Makefile
 	examples/Makefile
 	vncterm/Makefile
-	classes/Makefile
-	classes/ssl/Makefile
 	libvncclient/Makefile
 	client_examples/Makefile
 	test/Makefile
--- a/Makefile.am
+++ b/Makefile.am
@@ -4,8 +4,8 @@ if WITH_X11VNC
 X11VNC=x11vnc
 endif
 
-SUBDIRS=libvncserver examples contrib libvncclient vncterm classes client_examples test $(X11VNC)
-DIST_SUBDIRS=libvncserver examples contrib libvncclient vncterm classes client_examples test
+SUBDIRS=libvncserver examples contrib libvncclient vncterm client_examples test $(X11VNC)
+DIST_SUBDIRS=libvncserver examples contrib libvncclient vncterm client_examples test
 
 bin_SCRIPTS = libvncserver-config
 
